CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Changed
 - add error handling for apply changes action
+- after clicking 'Apply changes' change workflow in such a way, that initially new job is attempted to be created immediately. Only if it is impossible, schedule it for the future. 
```

Comment:
after clicking 'Apply changes' workflow is initially attempting to create new job immediately, if it is impossible, schedule it for the future

Reply:
done